modules/dit/module.json

```diff
@@ -2,7 +2,7 @@
 	"id": "dit",
 	"name": "dit",
 	"title": "Dead in Thay",
-	"description": "<blockquote>Adventure for Levels 9-11</blockquote><h2>ABOUT THE ORIGINAL</h2><p><em>Dead in Thay</em>, by Scott Fitzgerald Gray, was originally produced in 2014 as an adventure for the D&amp;D Encounters organized play program. At that time, it also served as a playtest for the rules that eventually became the fifth edition of the game. This version of the adventure has been modified for home play.</p><p>Featuring an immense and lethal dungeon, the adventure is a tribute to <em>Tomb of Horrors</em>, <em>Ruins of Undermountain</em>, and other killer dungeons from the game's history.</p>",
+	"description": "<blockquote>Adventure for Levels 9-11</blockquote><h3>ABOUT THE ORIGINAL</h3><p><em>Dead in Thay</em>, by Scott Fitzgerald Gray, was originally produced in 2014 as an adventure for the D&amp;D Encounters organized play program. At that time, it also served as a playtest for the rules that eventually became the fifth edition of the game. This version of the adventure has been modified for home play.</p><p>Featuring an immense and lethal dungeon, the adventure is a tribute to <em>Tomb of Horrors</em>, <em>Ruins of Undermountain</em>, and other killer dungeons from the game's history.</p>",
```

Comment:
The URL for this module wasn't updated by the script and 404's. I think the correct one is `https://www.dndbeyond.com/sources/dnd/tftyp/a5/dead-in-thay` ?

Reply:
On production the sourceSlug points to `https://www.dndbeyond.com/sources/dnd/tftyp`, which looks more correct.